src/diagnostics/codeactionprovider.ts

```diff
@@ -21,7 +21,8 @@ class PorterCodeActionProvider implements vscode.CodeActionProvider {
             return [];
         }
 
-        const actions = linters.map((l) => l.fixes(document, manifest, context.diagnostics));
+        const diagnostics = context.diagnostics;
+        const actions = linters.map((l) => l.fixes(document, manifest, diagnostics.flat()));
```

Comment:
❤️🙏💡 This was done because of the readonly property issue: https://stackoverflow.com/questions/53412934/disable-allowing-assigning-readonly-types-to-non-readonly-types 

Please feel free to suggest any thing else if anyone has thoughts.